Deployment Guides/Provisioning-Islandora-on-Ubuntu.md

```diff
@@ -120,280 +129,279 @@ cat ~/islandora-install.properties
+#### x264 <a id="x264"></a>
 
-<code>checkinstall --pkgname=yasm --pkgversion="1.2.0" --backup=no --deldoc=yes --fstrans=no --default</code>
+`cd ~/ffmpeg-source`
 
-_x264_  
+`git clone --depth 1 git://git.videolan.org/x264.git`
 
-<code>cd ~/ffmpeg-source</code>
+`cd x264 `
 
-<code>git clone --depth 1 git://git.videolan.org/x264.git</code> 
+`./configure --enable-static --enable-shared`
 
-<code>cd x264</code>  
+`make`
 
-<code>./configure --enable-static --enable-shared</code>  
+`checkinstall --pkgname=x264 --pkgversion="3:$(./version.sh | awk -F'[" ]' '/POINT/{print $4"+git"$5}')" --backup=no --deldoc=yes  --fstrans=no --default`
 
-<code>make</code>  
+`ldconfig`
 
-<code>checkinstall --pkgname=x264 --pkgversion="3:$(./version.sh | awk -F'[" ]' '/POINT/{print $4"+git"$5}')" --backup=no --deldoc=yes  --fstrans=no --default</code>  
+`cd ~/ffmpeg-source`
 
-<code>ldconfig</code>  
+#### aac  <a id="aac"></a>
 
-<code>cd ~/ffmpeg-source</code>
+`git clone --depth 1 git://github.com/mstorsjo/fdk-aac.git`  
 
-_aac_  
+`cd fdk-aac`
 
-<code>git clone --depth 1 git://github.com/mstorsjo/fdk-aac.git</code>  
+`autoreconf -fiv`
 
-<code>cd fdk-aac</code>  
+`./configure --disable-shared`
 
-<code>autoreconf -fiv</code>  
+`make`
 
-<code>./configure --disable-shared</code>  
+`checkinstall --pkgname=fdk-aac --pkgversion="$(date +%Y%m%d%H%M)-git" --backup=no --deldoc=yes --fstrans=no --default`
 
-<code>make</code>  
+#### libvpx <a id="libvpx"></a>
 
-<code>checkinstall --pkgname=fdk-aac --pkgversion="$(date +%Y%m%d%H%M)-git" --backup=no --deldoc=yes --fstrans=no --default</code>
+`cd ~/ffmpeg-source`  
 
-_libvpx_ 
- 
-<code>cd ~/ffmpeg-source</code>  
+`git clone https://chromium.googlesource.com/webm/libvpx.git`  
 
-<code>git clone https://chromium.googlesource.com/webm/libvpx.git</code>  
+`cd libvpx`
 
-<code>cd libvpx</code>  
+`./configure --disable-examples --disable-unit-tests`
 
-<code>./configure --disable-examples --disable-unit-tests</code>  
+`make`  
 
-<code>make</code>  
+`checkinstall --pkgname=libvpx --pkgversion="1:$(date +%Y%m%d%H%M)-git" --backup=no --deldoc=yes --fstrans=no --default`
 
-<code>checkinstall --pkgname=libvpx --pkgversion="1:$(date +%Y%m%d%H%M)-git" --backup=no --deldoc=yes --fstrans=no --default</code>
+#### opus  <a id="opus"></a>
 
-_opus_  
+`cd ~/ffmpeg-source`  
 
-<code>cd ~/ffmpeg-source</code>  
+`git clone --depth 1 git://git.xiph.org/opus.git`  
 
-<code>git clone --depth 1 git://git.xiph.org/opus.git</code>  
+`cd opus`  
 
-<code>cd opus</code>  
+`./autogen.sh`  
 
-<code>./autogen.sh</code>  
+`./configure --disable-shared`  
 
-<code>./configure --disable-shared</code>  
+`make`  
 
-<code>make</code>  
+`checkinstall --pkgname=libopus --pkgversion="$(date +%Y%m%d%H%M)-git" --backup=no --deldoc=yes --fstrans=no --default  `
 
-checkinstall --pkgname=libopus --pkgversion="$(date +%Y%m%d%H%M)-git" --backup=no --deldoc=yes --fstrans=no --default  
-</code>
+#### ffmpeg <a id="ffmpeg2"></a>
```

Comment:
The other ffmpeg section. Different from the first, but labeled the same. Deliberate or duplicate?